src/LobbyServer.hs

```diff
@@ -98,7 +107,7 @@ createGame remoteGames remoteClientList maxPlayers = do
 
   liftIO $ CC.modifyMVar_ games $ \gs ->
     case maybeClientEntry of
-        Just c  -> return $ (uuidStr, GameData [c] "GameName" maxPlayers) : gs
+        Just c  -> return $ (uuidStr, GameData [c] "GameName" maxPlayers empty) : gs
```

Comment:
Can be replaced with:
`return $ maybe gs (\c -> (uuidStr, GameData [c] "GameName" maxPlayers empty) : gs) maybeClientEntry`